calabash/Classes/FranklyServer/Routes/LPBackdoorRoute.m

```diff
@@ -29,40 +34,41 @@ - (NSDictionary *) JSONResponseForMethod:(NSString *) method URI:(NSString *) pa
     selectorName = [selectorName stringByAppendingString:@":"];
   }
 
+  id argument = [data objectForKey:@"arg"];
+  if (!argument) {
+    LPLogError(@"Expected data dictionary to contain an 'arg' key");
```

Comment:
Hey... wait... 

What's this [`__self__` keyword](https://github.com/calabash/calabash-ios-server/blob/develop/calabash/Classes/Invoker/LPInvoker.m#L651)? Is this an Obj-C thing or something we're introducing?

If it's the latter, could we not have a `@"__nil__"` ?